### ISSUE TITLE
Questions about training/testings set

### ISSUE THREAD

**Issue by ziruiw-dev** [sitzikbs/Nesti-Net#2]
Hi @sitzikbs,

Thanks for this work and releasing code. I have two questions regards your paper. 

1. In section 4.1 Datasets, you mentioned the PCPNet test dataset has 22 shapes. I downloaded the PCPNet dataset from the PCPNet project page and found there are only 19 shapes in the test set, and 3 shapes in the validation set and the validation set is a part of the test set, so if I understand correctly, there are only 19 shapes in the test set?

2. In section 4.2 Training details, you mentioned you trained with 32768 (1024 samples x 32 shapes) random subsets. Does that mean you count 8 shapes with 4 noise levels as 32 shapes?

Best,
Zirui

**Comment by sitzikbs** [comment]
We used exactly the same dataset and train/test split and setup as PCPNet. 

To answer your questions: 
1. You are correct. only 19 shapes. 15 + 3 analytic +1 boxy_smooth, just like in PCPNet. 
2. Yes, 8 shapes with 4 noise levels = 32 shapes.

**Comment by ziruiw-dev** [comment]
Hi @sitzikbs, Thanks for your reply. Best, Zirui

**Comment by ziruiw-dev** [comment]
A follow-up question, I noticed that you train with noise level 0.00125, 0.006 and 0.012 and I think none of them exists in the PCPNet dataset. May I ask did you augment those noise levels by yourself?

**Comment by sitzikbs** [comment]
Which file are you referring to?  

I trained with the same data they provided. I think these values correspond to their low, medium and high noise levels. All are a percentage of the bounding box.

**Comment by ziruiw-dev** [comment]
Thanks for replying.

When I look at the PCPNet dataset, for one shape, the xyz data in their dataset look like this:
- noise-free
    - armadillo100k.xyz
- gradient noise
    - armadillo100k_ddist_minmax.xyz
- striped noise
    - armadillo100k_ddist_minmax_layers.xyz
- brown noise
    - armadillo100k_noise_brown_1.00e-01.xyz
    - armadillo100k_noise_brown_1.00e-02.xyz
    - armadillo100k_noise_brown_2.00e-02.xyz
    - ...
- white noise
    - armadillo100k_noise_white_1.00e-02.xyz
    - armadillo100k_noise_white_2.00e-02.xyz
    - armadillo100k_noise_white_3.00e-02.xyz
    - ...

so I am not sure what file do you refer to when your use noise standard deviation sigma = 0.00125 or sigma = 0.006? The first image below is from your Nesti-Net paper, the second image is the PCPNet dataset. 

Many thanks.

![image](https://user-images.githubusercontent.com/9391023/69065247-5d0ed480-0a17-11ea-8482-7e511f48eb62.png)

![image](https://user-images.githubusercontent.com/9391023/69065628-f6d68180-0a17-11ea-8465-b5fafa8c74b1.png)

**Comment by ziruiw-dev** [comment]
Ok... I think I understood what you mean. The PCPNet dataset provides a trainset_whitenoise.txt file that specifies they trained with white noise level 1e-2, 5e-2, 1e-1.

Thanks a lot.
